src/main/resources/application.conf

```diff
@@ -260,25 +260,25 @@ spray.can {
 openolitor {
   run-proxy-service: true,
   port: 9003,
-  mandanten: ["csa1"],
-  csa1: {
-    name: "csa1",
+  mandanten: ["try"],
```

Comment:
Did you change the configuration in the openoitor-docker-compose project as well?

Reply:
No. This is a mistake and I gonna revert. Thanks for pointing out.